.github/workflows/build-pr.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and test PR (changed APIs only)
+
+on: [pull_request]	
+
+jobs:	
+
+  build:	
+    runs-on: ubuntu-18.04	
+    env:	
+      DOTNET_NOLOGO: true	
+    strategy:	
+      matrix:	
+        regex: ["'Google\\.Cloud\\.[A-L].*'", "'Google\\.Cloud\\.[M-Z].*'", "'!Google\\.Cloud'"]	
+    steps:	
+    - uses: actions/checkout@v2	
+      with:	
+        submodules: true	
+        fetch-depth: 100	
+
+    # We need just the .NET Core 2.1 runtime for testing    	
```

Comment:
We need .Net Core 3 for Diagnostics.AspNetCore3

Reply:
The comment is meant to indicate that we don't need .NET Core 2.1 for building - only for testing. (It's "just the runtime".)
Will leave now rather than making a more complex comment.

Reply:
Updated the comment in the end :)